lib/tools/bundletool.js

```diff
@@ -0,0 +1,114 @@
+  let installArgs = [
+    '-jar', path.resolve(this.helperJarPath, 'bundletool.jar'), 'install-apks',
+    '--apks', apks, '--device-id', deviceId
+  ];
+
+  if (!_.isEmpty(modules)) {
+    installArgs.push('--modules', modules.join(','));
+  }
+
+  log.info(`Install apks with: ${installArgs}`);
+  try {
+    return await exec(getJavaForOs(), installArgs);
+  } catch (e) {
+    throw new Error(`Failed to install apks. Original error ${e.message}`);
+  }
+};
+
+/**
+ * Check build version
+ *
+ * @param {string} bundle - The full path to the local `aab` file.
+ * @param {string} deviceId - The device serial to install the apks to.
+ * @param {string} output - List of modules to be installed (defaults to all of them).
+ *                           Note that the dependent modules will also be installed.
+ *                           Ignored if the device receives a standalone APK.
+ * @param {object} buildApksOpts - Options
+ * @property {string} ks - Path to the keystore that should be used to sign the
+ *                         generated APKs. If not set, the APKs will not be signed. If set, the
+ *                         flag 'ks-key-alias' must also be set.
+ * @property {string} ksKeyAlias - Alias of the key to use in the keystore to sign the generated APKs.
+ * @property {string} ksPass - Alias of the key to use in the keystore to sign the generated APKs.
+ *                             Password of the keystore to use to sign the generated APKs.
+ *                             If provided, must be prefixed with either 'pass:' (if the password
+ *                             is passed in clear text, e.g. 'pass:qwerty') or 'file:' (if the password
+ *                             is the first line of a file, e.g. 'file:/tmp/myPassword.txt'). If this
+ *                             flag is not set, the password will be requested on the prompt.
+ * @property {string} overwrite If set, any previous existing output will be overwritten.                           --overwrite
+ * @property {string} otherOptions - Add additional option
+ * @return {string} A path to apks
+ * @throws {Error} If building apks fails.
+ */
+bundletool.buildApks = async function (bundle, deviceId, output, buildApksOpts = {}) {
+  let buildApksArgs = [
+    '-jar', path.resolve(this.helperJarPath, 'bundletool.jar'), 'build-apks',
+    '--bundle', bundle,
+    '--output', output,
+    '--connected-device', '--device-id', deviceId
+  ];
+
+  if (buildApksOpts.ks && buildApksOpts.ksKeyAlias) {
+    buildApksArgs.push('--ks', buildApksOpts.ks);
+    buildApksArgs.push('--ks-key-alias', buildApksOpts.ksKeyAlias);
+  }
+
+  if (buildApksOpts.ksPass) {
+    buildApksArgs.push('--ks-pass', buildApksOpts.ksPass);
+  }
+
+  if (buildApksOpts.overwrite) {
+    buildApksArgs.push('--overwrite');
+  }
+
+  if (buildApksOpts.otherOptions) {
+    buildApksArgs.push(buildApksOpts.otherOptions);
```

Comment:
will it work this way? shouldn't otherOptions be an array, which is them merged to buildApksArgs?